ui/yarn.lock

```diff
@@ -528,15 +528,15 @@ pino-abstract-transport@^1.2.0:
     readable-stream "^4.0.0"
     split2 "^4.0.0"
 
-pino-http@^10.4.0:
-  version "10.4.0"
-  resolved "https://registry.yarnpkg.com/pino-http/-/pino-http-10.4.0.tgz#a97911138fd5170e6da92ae8882c2a6357e84115"
-  integrity sha512-vjQsKBE+VN1LVchjbfLE7B6nBeGASZNRNKsR68VS0DolTm5R3zo+47JX1wjm0O96dcbvA7vnqt8YqOWlG5nN0w==
+pino-http@^10.5.0:
```

Comment:
📋 **Yarn Lock Consistency**: Good to see yarn.lock is properly updated alongside package-lock.json. Both package managers now point to the same pino-http@10.5.0 version with matching integrity hashes.